test/clustering_quality.jl

```diff
@@ -0,0 +1,58 @@
+using Test
+using Clustering, Distances
+using OffsetArrays
+
+@testset "clustering_quality()" begin
+
+    # test data with 4 clusters
+
+    Y = [-2 4; 2 4; 2 1; 3 0; 2 -1; 1 0; 2 -4; -2 -4; -2 1; -1 0; -2 -1; -3 0]
+    C = [0 4; 2 0; 0 -4; -2 0]
+    A = [1, 1, 2, 2, 2, 2, 3, 3, 4, 4, 4, 4]
+    W = [
+        1 0 0 0
+        1 0 0 0
+        0 1 0 0
+        0 1 0 0
+        0 1 0 0
+        0 1 0 0
+        0 0 1 0
+        0 0 1 0
+        0 0 0 1
+        0 0 0 1
+        0 0 0 1
+        0 0 0 1
+    ]
+
+    # visualisation of the data
+    # using Plots
+    # scatter(Y[:,1],Y[:,2],
+    #     axisratio = :equal,
+    #     #seriescolor = palette(default)[A],
+    # )
+    # scatter!(C[:,1],C[:,2],
+    #     marker = :square,
+    #     label = "cluster centers",  
+    # )
+
+    @testset "input checks" begin
```

Comment:
can you test for incorrect `quality_index` for each of the `clustering_quality()` variant here?